README.md

```diff
@@ -42,8 +42,6 @@ octokit.createPullRequest({
 
 ## Todos
 
-- **Creates a fork if necessary**  
-  If the authenticated user does not have write access to the repository then create a fork first.
```

Comment:
Maybe add it to features on top of the README?

Reply:
I moved this line to the feature section 🚀